### ISSUE TITLE
Character Customization Change Appearance bug(?)

### ISSUE THREAD

**Issue by Hyrenn** [dalaranwow/dalaran-wow#5140]
![screenshot 1](https://user-images.githubusercontent.com/34147118/33464276-0ede7ebe-d607-11e7-8df9-7a1aa62c0686.png)
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
In the "Character Customization" tab on Dalaran-wow, I can only change the appearance of characters on Kirin Tor server, not on Algalon.
**Expected behaviour**: Tell us what should happen instead
Characters from Algalon should be available in this tab
**Steps to reproduce the problem**: 

1. Create character on Algalon and Kirin Tor
2. Go to Character Customization on Dalaran-wow
3. See option to change appearance only for character on Kirin Tor

**Include proofs for this behaviour**
Proof in comments
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Hyrenn** [comment]
![screenshot 1](https://user-images.githubusercontent.com/34147118/33464310-377352e6-d607-11e7-9977-f9fd910947c8.png)
https://user-images.githubusercontent.com/34147118/33464276-0ede7ebe-d607-11e7-8df9-7a1aa62c0686.png

**Comment by Crispybuilt** [comment]
Hello,

Bug Tracker lists only in-game issues and not the website features.

Invalid Report
Closing up
